### PR TITLE
feat(files): 3D Mace

### LINKED ISSUE #46: [3D] 3D Mace
Check if is possible in the latest update preferably attachables

### PR DESCRIPTION
**Pull request by MPAD2277**
Add/update files for new tweaks, 3D Mace:
- Add models, textures, animation, and attachable file for 3D Mace.
- Update packs.json.

Preview in first person:
Screenshot_20240704-144339
Preview in third person (from @PAPISOP):
image

Resolves #46 

By checking the following boxes with an X, you ensure that:

- [x] The pack was tested ingame in at least one device.
- [x] The pack is an existing BT pack, is a missing pack from VT or is an accepted pack/change in a discussion.
- [x] The pack code follows the style guide.
- [x] The commits follow the contribution guidelines.
- [x] The PR follows the contribution guidelines.

- [ ] (Optional) Tested in Windows
- [x] (Optional) Tested in Android
- [ ] (Optional) Tested in iOS
- [ ] (Optional) Tested in any console
- [ ] (Optional) Tested in BDS

### CODE REVIEW

**Review by NSPC911 — APPROVED** [review]
Nice

**Review by SquatchHunter — APPROVED** [review]
lgtm
only suggestion would be move 3d Sun & Moon and this pack to a new 3d Category

**Comment by MPAD2277** [comment]
> lgtm only suggestion would be move 3d Sun & Moon and this pack to a new 3d Category

Its kinda awkward if the 3d category only have 2 tweaks inside

**Comment by SquatchHunter** [comment]
It may look weird, it will eventually have a 3rd lol

We have discussed 3d blocks being custom blocks and basically just "vanity" alternates but it will be addons not rp, we *should* be able to make a few more 3d packs in the interim

**Comment by drav0011** [comment]
- Check contributing guidelines for file names
https://github.com/BedrockTweaks/Files/blob/devel/CONTRIBUTING.md
- The formatting in the attachable file (should be tab size 4)
- And it is missing the item icon (Note: in java it uses the item model, here we would need to replace the "flat" texture)
image

**Comment by MPAD2277** [comment]
> * Check contributing guidelines for file names
>   https://github.com/BedrockTweaks/Files/blob/devel/CONTRIBUTING.md
> * The formatting in the attachable file (should be tab size 4)
> * And it is missing the item icon (Note: in java it uses the item model, here we would need to replace the "flat" texture)
>   image

Done 👍

File: resource_packs/files/aesthetic/3d_mace/textures/items/mace1.png
Comment (drav0011) [inline resource_packs/files/aesthetic/3d_mace/textures/items/mace1.png:1]: this file what is it for, is it needed?

**Comment by drav0011** [comment]
> only suggestion would be move 3d Sun & Moon and this pack to a new 3d Category

Also note about this, the category will be created, world of color only contains one pack (without having in account subcategories), it's ok to not have it in this pr and make it in a separate one with #132

File: resource_packs/files/aesthetic/3d_mace/textures/items/mace1.png
Comment (MPAD2277) [reply resource_packs/files/aesthetic/3d_mace/textures/items/mace1.png:1]: Oh why is it there? It's supposed to not be there. Lemme delete it

**Comment by MPAD2277** [comment]
> > only suggestion would be move 3d Sun & Moon and this pack to a new 3d Category
> 
> Also note about this, the category will be created, world of color only contains one pack (without having in account subcategories), it's ok to not have it in this pr and make it in a separate one with #132

So, i need to move it to the new category?

**Review by drav0011 — APPROVED** [review]
lgtm

**Comment by drav0011** [comment]
> So, i need to move it to the new category?

No need I'll handle it before release in a moment

**Comment by MPAD2277** [comment]
Yo, thanks for the merge

**Pull request merged**